### ISSUE TITLE
Current example in the README is not much of use

### ISSUE THREAD

**Issue by kaushalyap** [meilisearch/gatsby-plugin-meilisearch#180]
### Other

Current example on README create index generated pages by path, this is less useful, why?

- Users does not want to search queries that match the slug, they want the pages where content matches with their queries.

I know what is shown in README is a dummy example, but it is worth to have real life example like product search. Example [using Medusa](https://docs.medusajs.com/add-plugins/meilisearch) would be great.

**Comment by bidoubiwa** [comment]
Hello @kaushalyap ! I agree with your comment, we should definitely provide a better example in the README! Initially we thought about creating an example following [this tutorial](https://www.gatsbyjs.com/docs/tutorial/part-5/), where the data inside the markdown would be indexed in Meilisearch.

If you are willing to create the example it would be awesome 🔥 No problem if you don't :)

**Comment by oluademola** [comment]
Hello @kaushalyap 

Thank you very much for your suggestion. I am not sure I completely understand your example with Medusa, and it is something we have not considered. However, this being a potential example in the README, the intention would be to create something clear and simple as the previously mentioned tutorial. Let me know your thoughts and if I'm missing anything. 

Thanks again.

**Comment by kaushalyap** [comment]
@oluademola I agree it should be simple and understandable by majority. So I would suggest going with a markdown page indexing example.
